samples/nrf9160/serial_lte_modem/src/slm_at_tcpip.c

```diff
@@ -8,67 +8,98 @@
 #include <stdio.h>
 #include <string.h>
 #include <net/socket.h>
+#include <modem_info.h>
+#include <net/tls_credentials.h>
 #include "slm_at_host.h"
 #include "slm_at_tcpip.h"
 
 LOG_MODULE_REGISTER(tcpip, CONFIG_SLM_LOG_LEVEL);
 
 #define INVALID_SOCKET	-1
+#define INVALID_SEC_TAG	-1
 #define TCPIP_MAX_URL	128
 
 /*
  * Known limitation in this version
  * - Multiple concurrent sockets
  * - Socket type other than SOCK_STREAM(1) and SOCK_DGRAM(2)
  * - IP Protocol other than TCP(6) and UDP(17)
- * - TCP/UDP server mode
+ * - TCP server accept one connection only
  * - Receive more than IPv4 MTU one-time
  * - IPv6 support
+ * - does not support proxy
  */
 
 /**@brief Socket operations. */
 enum slm_socket_operation {
 	AT_SOCKET_CLOSE,
-	AT_SOCKET_OPEN
+	AT_SOCKETOPT_GET = AT_SOCKET_CLOSE,
```

Comment:
IMO it would be cleaner to have a separate enum for that, `slm_socketopt_operation`, right num it's confusing to see the enum values being redefined.

Reply:
The only reason to do like this is to use 0 for Close and Get, 1 for Open and Set in the AT command interface

Reply:
Updated